README.md

```diff
@@ -47,7 +47,9 @@
 
 ### Python
 
-- [fastparquet](https://github.com/dask/fastparquet/) - A Python implementation of the Parquet columnar file format. 
+- [fastparquet](https://github.com/dask/fastparquet/) - A Python implementation of the Parquet columnar file format.
+- [petastorm](https://github.com/uber/petastorm) - Petastorm is a library enabling the use of Parquet storage from Tensorflow, Pytorch, and other Python-based ML training frameworks.
```

Comment:
This one is based on PyArrow and PySpark, but as it's specifically using Parquet files, I think it makes a lot of sense to include it.

Reply:
You are totally right ! I just looked at the last commit date... sorry